lib/utils.sh

```diff
@@ -206,3 +206,29 @@ get_asdf_config_value() {
         get_asdf_config_value_from_file $default_config_path $key
     fi
 }
+
+asdf_repository_url() {
+  echo "https://github.com/doughsay/asdf-plugins.git"
+}
+
+initialize_or_update_repository() {
+  local repository_url=$(asdf_repository_url)
+  local repository_path=$(asdf_dir)/repository
+
+  if [ -d $repository_path ]; then
```

Comment:
This will probably fail if `asdf_dir` contains spaces. Adding quotes should do the job.